gammapy/astro/darkmatter/spectra.py

```diff
@@ -291,23 +298,21 @@ class DarkMatterDecaySpectralModel(SpectralModel):
         See `PrimaryFlux.channel_registry` for more.
     scale : float
         Scale parameter for model fitting
-    jfactor : `~astropy.units.Quantity`
-        Integrated J-Factor needed when `~gammapy.modeling.models.PointSpatialModel`
-        is used.
+    dfactor : `~astropy.units.Quantity`
+        Integrated D-Factor for model fitting.
     z: float
         Redshift value.
 
     Examples
     --------
-    This is how to instantiate a `DarkMatterAnnihilationSpectralModel` model::
+    This is how to instantiate a `DarkMatterDecaySpectralModel` model::
 
         >>> import astropy.units as u
         >>> from gammapy.astro.darkmatter import DarkMatterDecaySpectralModel
 
         >>> channel = "b"
         >>> massDM = 5000*u.Unit("GeV")
-        >>> jfactor = 3.41e19 * u.Unit("GeV cm-2")
-        >>> modelDM = DarkMatterDecaySpectralModel(mass=massDM, channel=channel, jfactor=jfactor)  # noqa: E501
+        >>> modelDM = DarkMatterDecaySpectralModel(mass=massDM, channel=channel)  # noqa: E501
```

Comment:
same here why removing the jfactor from the example ? this pattern should still work